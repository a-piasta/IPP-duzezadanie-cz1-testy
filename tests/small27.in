addRoad;xjmjoqlcchvfphfncqvqqsj;seqexvudcbg;2;-5
extendRoute;50;seqexvudcbg
addRoad;clltrdgmyvtlioavrkoblwoi;glmezwvyikjyauaebvjrco;9;11
addRoad;vzmnppiivbos;gztjlvmmleal;26;-14
removeRoad;vzmnppiivbos;gztjlvmmleal
getRouteDescription;96
removeRoad;xjmjoqlcchvfphfncqvqqsj;seqexvudcbg
removeRoad;glmezwvyikjyauaebvjrco;clltrdgmyvtlioavrkoblwoi
removeRoad;xjmjoqlcchvfphfncqvqqsj;seqexvudcbg
newRoute;130;xjmjoqlcchvfphfncqvqqsj;seqexvudcbg
repairRoad;hgbxxtjzbwngxkdxungsoc;seqexvudcbg;-5
repairRoad;gztjlvmmleal;azmbaipihbm;11
extendRoute;130;glmezwvyikjyauaebvjrco
addRoad;glmezwvyikjyauaebvjrco;dxyczkhavnqkrr;28;6
repairRoad;seqexvudcbg;dxyczkhavnqkrr;-12
removeRoad;vzmnppiivbos;gztjlvmmleal
newRoute;328;glmezwvyikjyauaebvjrco;glmezwvyikjyauaebvjrco
repairRoad;dxyczkhavnqkrr;clltrdgmyvtlioavrkoblwoi;-9
getRouteDescription;328
addRoad;vzmnppiivbos;clltrdgmyvtlioavrkoblwoi;3;-14
addRoad;clltrdgmyvtlioavrkoblwoi;clltrdgmyvtlioavrkoblwoi;18;1
repairRoad;gztjlvmmleal;seqexvudcbg;-10
extendRoute;130;glmezwvyikjyauaebvjrco
repairRoad;gztjlvmmleal;dxyczkhavnqkrr;-15
repairRoad;ixajdbleoneyepgfbmaazpqoaajb;ixajdbleoneyepgfbmaazpqoaajb;3
getRouteDescription;130
addRoad;xjmjoqlcchvfphfncqvqqsj;seqexvudcbg;4;-14
extendRoute;328;azmbaipihbm
addRoad;dxyczkhavnqkrr;vzmnppiivbos;17;2
newRoute;716;azmbaipihbm;hgbxxtjzbwngxkdxungsoc
addRoad;gztjlvmmleal;azmbaipihbm;13;-5
newRoute;453;dxyczkhavnqkrr;vzmnppiivbos
extendRoute;453;dxyczkhavnqkrr
addRoad;glmezwvyikjyauaebvjrco;hgbxxtjzbwngxkdxungsoc;21;15
addRoad;vzmnppiivbos;vzmnppiivbos;6;-6
newRoute;652;seqexvudcbg;hgbxxtjzbwngxkdxungsoc
addRoad;dxyczkhavnqkrr;ixajdbleoneyepgfbmaazpqoaajb;24;-9
newRoute;300;gztjlvmmleal;ixajdbleoneyepgfbmaazpqoaajb
extendRoute;300;azmbaipihbm
removeRoad;clltrdgmyvtlioavrkoblwoi;glmezwvyikjyauaebvjrco
repairRoad;dxyczkhavnqkrr;ixajdbleoneyepgfbmaazpqoaajb;-10
extendRoute;130;azmbaipihbm
extendRoute;453;ixajdbleoneyepgfbmaazpqoaajb
getRouteDescription;328
extendRoute;453;dxyczkhavnqkrr
extendRoute;130;vzmnppiivbos
removeRoad;vzmnppiivbos;clltrdgmyvtlioavrkoblwoi
addRoad;clltrdgmyvtlioavrkoblwoi;dxyczkhavnqkrr;2;0
addRoad;clltrdgmyvtlioavrkoblwoi;seqexvudcbg;20;2
extendRoute;652;gztjlvmmleal
addRoad;dxyczkhavnqkrr;gztjlvmmleal;9;-12
repairRoad;clltrdgmyvtlioavrkoblwoi;ixajdbleoneyepgfbmaazpqoaajb;10
getRouteDescription;716
repairRoad;glmezwvyikjyauaebvjrco;clltrdgmyvtlioavrkoblwoi;15
removeRoad;dxyczkhavnqkrr;glmezwvyikjyauaebvjrco
extendRoute;328;clltrdgmyvtlioavrkoblwoi
newRoute;566;glmezwvyikjyauaebvjrco;xjmjoqlcchvfphfncqvqqsj
repairRoad;xjmjoqlcchvfphfncqvqqsj;clltrdgmyvtlioavrkoblwoi;-11
getRouteDescription;130
repairRoad;gztjlvmmleal;seqexvudcbg;4
