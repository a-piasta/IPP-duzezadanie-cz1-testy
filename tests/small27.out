1: TAK
2: NIE
3: TAK
4: TAK
5: TAK
6: 
7: TAK
8: TAK
9: NIE
10: NIE
11: NIE
12: NIE
13: NIE
14: TAK
15: NIE
16: NIE
17: NIE
18: NIE
19: 
20: TAK
21: NIE
22: NIE
23: NIE
24: NIE
25: NIE
26: 
27: TAK
28: NIE
29: TAK
30: NIE
31: TAK
32: TAK
33: NIE
34: TAK
35: NIE
36: NIE
37: TAK
38: NIE
39: NIE
40: NIE
41: NIE
42: NIE
43: TAK
44: 
45: NIE
46: NIE
47: TAK
48: NIE
49: TAK
50: NIE
51: TAK
52: NIE
53: 
54: NIE
55: TAK
56: NIE
57: NIE
58: NIE
59: 
60: NIE
